src/pages/Vault/modals/WithdrawFromVestingVaultModal/Withdraw/index.tsx

```diff
@@ -0,0 +1,120 @@
+/** @jsxImportSource @emotion/react */
+import React, { useContext, useMemo } from 'react';
+import BigNumber from 'bignumber.js';
+import isBefore from 'date-fns/isBefore';
+
+import TEST_IDS from 'constants/testIds';
+import { AuthContext } from 'context/AuthContext';
+import { TOKENS } from 'constants/tokens';
+import { TokenId } from 'types';
+import { getToken } from 'utilities';
+import useConvertWeiToReadableTokenString from 'hooks/useConvertWeiToReadableTokenString';
+import { useTranslation } from 'translation';
+import { useGetXvsVaultWithdrawalRequests, useExecuteWithdrawalFromXvsVault } from 'clients/api';
+import { ConnectWallet, Spinner, LabeledInlineContent, PrimaryButton } from 'components';
+import { useStyles } from './styles';
+
+export interface WithdrawProps {
+  stakedTokenId: TokenId;
+  poolIndex: number;
+  handleClose: () => void;
+}
+
+const Withdraw: React.FC<WithdrawProps> = ({ stakedTokenId, poolIndex, handleClose }) => {
```

Comment:
This component doesn't strictly follow our container patter

Reply:
Indeed!